test/puppetlabs/puppetdb/http/reports_test.clj

```diff
@@ -124,6 +124,30 @@
                                ["=" "certname" (:certname basic)]])
        #{(select-keys basic [:hash :certname :transaction_uuid])}))))
 
+(deftestseq query-metrics
+  [[version endpoint] [[:v4 (partial format "/v4/reports/%s/metrics")]]]
+
+  (let [basic         (:basic reports)
+        report-hash   (:hash (store-example-report! basic (now)))
+        basic (assoc basic :hash report-hash)]
```

Comment:
these are indented inconsistently. I think we've settled on not aligning anything.